Makefile

```diff
@@ -181,7 +181,13 @@ lint: tools verify-modules
 			$(MAKE) lint || exit 1; \
 			cd $$working_dir; \
 		fi; \
-	done
+	done; \
+	CHECK=$$(grep -r --include="*.go" ioutil ./); \
```

Comment:
Would it be possible to add [the `ioutilDeprecated` check](https://go-critic.com/overview.html#ioutildeprecated) to our golangci-lint configuration vs writing our own check?

Reply:
Thanks! Yeah, I've tried poking around it a bit and really have no idea why it's not failing today. According to the config, it should be running that check and flagging it. And I vaguely remember it actually doing so either here or in `tanzu-framework`, but it appears neither one is catching it anymore.